examples/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+rucio-integration-register -r /rucio/disks/xrd1/rucio -c HSC/runs/RC2/w_2023_32/DM-40356/20230814T170253Z -t visitSummary -d rubin_dataset -C register_config.yaml
```

Comment:
Are there going to be more rucio commands? If there are we should consider click and subcommands and renaming to something like `lsst-rucio register ...`

Reply:
This is the only command we have planned.